src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
+
+#footer-component {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+    background: #FFFFFF;
+    margin: 0px auto 0px auto;
+    bottom:0; 
+}
+
+#become-nanny {
+    width: 473.8px;
+    height: 24px;
+    margin: 0 0 20px;
+    font-family: Inter;
+    font-size: 28px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 0.86;
```

Comment:
I always add the line-height in rem. 0.86 means that the line-height is equals to: `font-size * 0.86` which results in a line-height smaller than the actual font-size.

![image](https://user-images.githubusercontent.com/32915144/104590352-f9e3f680-5649-11eb-9827-5100ab5dafdd.png)